assets/js/util/sort-object-map-by-key.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * sortObjectMapByKey utiltity function.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Sorts an object map based on numberic order/priority key.
+ *
+ * @param {Object} objectList  Object containg objects to be sorted
+ * @param {string} priorityKey The object key to use for matching.
+ *
```

Comment:
Should this empty line be here? 🤷

Reply:
There shouldn't be empty lines between param and return, only above the param/return group (there should be a `@since` annotation in here, which _should_ be separated by a blank line).